src/Command/Cron/AskFeedbackUsagerCommand.php

```diff
@@ -49,21 +50,80 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $totalRead = 0;
-        $io = new SymfonyStyle($input, $output);
+        $this->io = new SymfonyStyle($input, $output);
+
+        // on récupère les signalements dont les deux derniers suivis sont des suivis techniques demande de feedback
+        // et dont le dernier suivi a plus de 30 jours
+        $signalementsIdsThirdRelance = $this->suiviRepository->findSignalementsThirdRelance();
+        $nbSignalementsThirdRelance = $this->sendMailToUsagers(
+            $input,
+            $signalementsIdsThirdRelance,
+            NotificationMailerType::TYPE_SIGNALEMENT_FEEDBACK_USAGER_THIRD
+        );
+        $this->io->success(sprintf(
+            '%s signalement(s) with two last suivis are technicals and last one older than '.Suivi::DEFAULT_PERIOD_INACTIVITY.' days',
```

Comment:
pour de l'anglais correct, remplacer par 
`signalement(s) for which the two last suivis are technicals and the last one is older than`

Reply:
Ah ah merci :D 
J'ai du mal à croire que l'anglais soit correct sur les autres !